infertrade/algos/community/allocations.py

```diff
@@ -136,6 +176,14 @@ def weighted_moving_averages(
             "github_permalink": "https://github.com/ta-oliver/infertrade/blob/b2cf85c28ed574b8c246ab31125a9a5d51a8c43e/infertrade/algos/community/allocations.py#L34"
         },
     },
+    "chande_kroll_crossover_strategy": {
+        "function": chande_kroll_crossover_strategy,
+        "parameters": {"allow_short_selling": False},
```

Comment:
We haven't tried bool parameters before so am surprised did not create any issues for schema check.

We could consider allowing short selling by default as it is easy for user to trivially truncate results if needed? We could create a decorator for example that capped the range of allocation between a and b that users could apply to constain any allocation algorithm.

Reply:
>  We could create a decorator for example that capped the range of allocation between a and b that users could apply to constain any allocation algorithm.

I'll put that in an issue.